README.md

```diff
@@ -746,6 +747,87 @@ See [javadocs](https://static.runelite.net/api/runelite-api/net/runelite/api/Gra
 
 </details>
 
+### Metadata:
+
+On login, Dink can submit a character summary containing data that spans multiple notifiers to a custom webhook handler (configurable in the `Advanced` section).
+
+<details>
+  <summary>JSON for Login Notifications:</summary>
+
+```json5
```

Comment:
Please include information about `collectionLog` not being always sent. This also sent `clanName` for me which is missing from here. I'd like for this to include that we delay this webhook a little from the actual login.